docs/Plugins/jira.md

```diff
@@ -140,30 +133,37 @@ To collect data, select `Advanced Mode` on the `Create Pipeline Run` page and pa
 ```json /blueprints/{blueprintId}
```

Comment:
Can we move the 'Advanced Mode' part to the `Configuration` section in the same hierarchy as 'Configure via Config UI'?

Reply:
Good idea!